docs/getting-started.md

```diff
@@ -1,18 +1,18 @@
 # Getting Started
 
-1. [Configuring Travis CI and GitHub Actions](#configuring-travis-ci)
+1. [Configuring GitHub Actions](#configuring-github-actions)
 1. [Local installation](#local-installation)
 1. [Making ORCA aware of your package](#making-orca-aware-of-your-package)
 1. [Running automated tests](#running-automated-tests)
     1. [PHPUnit](#phpunit)
     1. [Nightwatch.js](#nightwatchjs)
     1. [Tagging/grouping](#tagginggrouping)
 
-## Configuring Travis CI and GitHub Actions
+## Configuring GitHub Actions
```

Comment:
Let's keep this generic--not product-specific--to minimize possible changes needed in the future.

```suggestion
## Configuring CI
```